### PR TITLE
Add collab book for hugging-face loading model.

### PR DESCRIPTION
**Pull request by radoondas**
Initial commit, which adds collab book for:
 - loading the model from hugging-face,
 - ingesting the Ingest pipeline,
 - indexing documents using the pipeline,
 - and performing a simple query using Python code

### CODE REVIEW

File: notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb [hunk 1]
```diff
@@ -0,0 +1,1109 @@
...
+      "Indexed 19717 blogs embeddings documents\n"
+     ]
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Query the dataset\n",
+    "The next step is to run a query to search for relevant blogs. The example query searches for `\"model_text\": \"when elasticsearch released vecor search\"` using the model we uploaded to Elasticsearch `sentence-transformers__msmarco-minilm-l-12-v3`.\n",
```
Comment (TattdCodeMonkey) [inline notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb:715]:
```suggestion
    "The next step is to run a query to search for relevant blogs. The example query searches for `\"model_text\": \"how to do image similarity search\"` using the model we uploaded to Elasticsearch `sentence-transformers__msmarco-minilm-l-12-v3`.\n",
```

File: notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb [hunk 2]
```diff
@@ -0,0 +1,1109 @@
...
+     "execution_count": 7
+    }
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Get blogs dataset\n",
+    "Download:\n",
+    "- The example blogs dataset is located in [GitHub](https://github.com/radoondas/flask-elastic-nlp/blob/main/embeddings/blogs/blogs-no-embeddings.json.zip)\n",
```
Comment (TattdCodeMonkey) [inline notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb:508]: we should update this link to the elasticsearch-labs repo

Comment (radoondas) [reply notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb:508]: Yes. I forgot to change this link as I already did in the code itself. I'll fix this.

File: notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb [hunk 3]
```diff
@@ -0,0 +1,1109 @@
...
+    "\n",
+    "response = es.search(\n",
+    "    index=INDEX_NAME,\n",
+    "    fields=source_fields,\n",
+    "    knn=query,\n",
+    "    source=False)\n",
+    "\n",
+    "print(response.body)\n",
+    "\n",
+    "results = pd.json_normalize(json.loads(json.dumps(response.body['hits']['hits'])))\n",
```
Comment (TattdCodeMonkey) [inline notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb:773]: isn't `json.loads(json.dumps())` redundant?

**Pull request merged**